model/riscv_pmp_control.sail

```diff
@@ -92,6 +92,21 @@ function pmpAddrRange(cfg: Pmpcfg_ent, pmpaddr: xlenbits, prev_pmpaddr: xlenbits
   }
 }
 
+/* Check mseccfg.MMl=1 and mseccfg.MMWP=0 and make execute false for Mmode for matching region not found*/
```

Comment:
This sentence doesn't make much sense, but don't just write out a copy of the implementation in English, document what it _means_

Reply:
Also you need a space before `*/`

Reply:
All comments are improved and formatting is done